lighthouse-cli/test/smokehouse/frontends/firehouse.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license Copyright 2020 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/**
+ * @fileoverview Smoke test runner.
+ * Used to test integrations that run Lighthouse within a browser (i.e. LR)
```

Comment:
```suggestion
 * Used to test integrations that run Lighthouse within a browser (i.e. LR, DevTools)
```

is that true? or is this LR specific?

Reply:
I will use it for DT too.